Cargo.toml

```diff
@@ -1,12 +1,15 @@
 [package]
 name = "prima_datadog"
-version = "0.1.6"
+version = "0.1.7"
 edition = "2018"
 description = "An opinionated library to share code and approach to Datadog logging in prima.it"
 license = "MIT"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[features]
+dev-null = []
```

Comment:
This is a v1. I'm working on tests; anyway.. do someone propose a better name for this feature? :)

Reply:
`noop` maybe?